#!/bin/sh
./manage.py runserver 16721
